src/scss/login.scss

```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;600;700&family=Source+Sans+Pro:wght@200;300;400;600;700&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Inter', sans-serif;
+    font-family: 'Source Sans Pro', sans-serif;
+}
+
+.login {
+    color: black;
+}
+form {
+    padding: 30px;
+    max-width: 30em;
+    margin: 0 auto;
+}
```

Comment:
A class or id should likely be used here instead of styling all form elements. Another may need to be used which has different styling, which would make debugging harder. If we move login.scss to a higher or lower load order either your work or someone elses could break

Reply:
This is also the case on the input elements. Though ussing SCSS here may help, by adding them into the form ruleset will make them only apply to it's children

```
.login-form {
   // login form styles
    $ > input[type=password] {
        // styles only apply to inputs of type password inside a .login-form element
    }
}
```

Reply:
change form selector to className= ".submit-login"
made ruleset for signin form